subprojects/core/src/main/java/org/gradle/api/internal/FeaturePreviews.java

```diff
@@ -28,7 +28,8 @@
     public enum Feature {
         IMPROVED_POM_SUPPORT(false),
         GRADLE_METADATA(true),
-        STABLE_PUBLISHING(false);
+        STABLE_PUBLISHING(false),
+        INCREMENTAL_ARTIFACT_TRANSFORMATIONS(true);
```

Comment:
Should this be on by default?

Reply:
The is that it shouldn't be on by default, right? If it would be on by default, I wouldn't need the feature preview at all. It would just always be on.

Or do you mean the `true` in brackets? That doesn't mean that it is on by default, it means that the feature flag still is active, i.e. has an effect. If the feature flag is inactive, then a deprecation warning is printed if you use it.

Reply:
Ah, thanks. I wish we used enums for these things.